src/components/Breadcrumb.vue

```diff
@@ -46,3 +46,14 @@ export default {
   mixins: [categoryMixin],
 };
 </script>
+
```

Comment:
Put `<-- eslint-disable -->` before each <i18n> tag to avoid having future errors on this, which should not be checked.

Reply:
@lauraluiz done!